packages/ledger/project.json

```diff
@@ -1,54 +1,55 @@
 {
-  "root": "packages/ledger",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "packages/ledger/src",
   "projectType": "library",
   "targets": {
     "build": {
-      "executor": "@nrwl/web:rollup",
-      "outputs": ["{options.outputPath}"],
+      "executor": "@nrwl/js:tsc",
```

Comment:
It looks like the build command for all packages has been changed to this builder `@nrwl/js:tsc` can you please revert it back to `@nrwl/web:rollup` we need to support ESM and CJS too.